File: app/src/main/java/com/raysk/intertec/ButtonNavigation.java
package com.raysk.intertec;

import android.os.Bundle;

import com.google.android.material.bottomnavigation.BottomNavigationView;

import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.widget.Toolbar;
import androidx.navigation.NavController;
import androidx.navigation.Navigation;
import androidx.navigation.ui.AppBarConfiguration;
import androidx.navigation.ui.NavigationUI;

import com.raysk.intertec.databinding.ActivityButtonNavigationBinding;

public class ButtonNavigation extends AppCompatActivity {

    private ActivityButtonNavigationBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        binding = ActivityButtonNavigationBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());

        BottomNavigationView navView = findViewById(R.id.nav_view);
        // Passing each menu ID as a set of Ids because each
        // menu should be considered as top level destinations.
        /*AppBarConfiguration appBarConfiguration = new AppBarConfiguration.Builder(
                R.id.datosGeneralesFragment)
                .build();*/
        NavController navController = Navigation.findNavController(this, R.id.nav_host_fragment_activity_button_navigation);
        //NavigationUI.setupActionBarWithNavController(this, navController);
        NavigationUI.setupWithNavController(binding.navView, navController);

    }

}

File: app/src/main/java/com/raysk/intertec/ButtonNavigation.kt
package com.raysk.intertec

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.navigation.Navigation.findNavController
import androidx.navigation.ui.NavigationUI.setupWithNavController
import com.google.android.material.bottomnavigation.BottomNavigationView
import com.raysk.intertec.util.update.Updates

class ButtonNavigation : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        setContentView(R.layout.activity_button_navigation)
        val navView = findViewById<BottomNavigationView>(R.id.nav_view)
        // Passing each menu ID as a set of Ids because each
        // menu should be considered as top level destinations.
        /*AppBarConfiguration appBarConfiguration = new AppBarConfiguration.Builder(
                R.id.datosGeneralesFragment)
                .build();*/
        val navController =
            findNavController(this, R.id.nav_host_fragment_activity_button_navigation)
        //NavigationUI.setupActionBarWithNavController(this, navController);
        setupWithNavController(navView, navController)

        val updates = Updates(this)
        updates.buscar()
    }
}